src/integrationTest/java/net/twasi/obsremotejava/test/manual/CallbackObservationTest.java

```diff
@@ -0,0 +1,91 @@
+package net.twasi.obsremotejava.test.manual;
+
+import net.twasi.obsremotejava.OBSRemoteController;
+import org.junit.jupiter.api.Test;
+
+public class CallbackObservationTest {
+
+  private final String obsAddress = "ws://localhost:4444";
+  private final String obsPassword = null;
+
+  @Test
+  void test() {
+    final OBSRemoteController controller = new OBSRemoteController(obsAddress, false, null);
+
+    if (controller.isFailed()) {
+      System.out.println("UPS DAS GET NET HÜLFEEE!");
```

Comment:
You might want the output english here ;)

Reply:
Was going to leave this one alone for now -- plan to replace it with testing the actual callbacks.